tests/gh-api-helpers.js

```diff
@@ -0,0 +1,191 @@
+const { Octokit } = require('octokit')
+const axios = require('axios')
+const AdmZip = require('adm-zip')
+
+const ref = process.env.GITHUB_HEAD_REF || process.env.GITHUB_REF
+const owner = process.env.GITHUB_REPOSITORY_OWNER
+const repo = process.env.GITHUB_REPOSITORY.split('/')[1]
+const workflow_id = 'integration.yml'
+
+const octokit = new Octokit({
+  auth: process.env.GITHUB_TOKEN,
+})
+
+const helpers = {
+  scheduleAndRun: async (scheduler_res_path, runName) => {
+    return await octokit.request(
+      'POST /repos/{owner}/{repo}/actions/workflows/{workflow_id}/dispatches',
+      {
+        owner,
+        repo,
+        workflow_id,
+        ref,
+        inputs: {
+          scheduler_res_path,
+          run_name: runName,
+        },
+        headers: {
+          'X-GitHub-Api-Version': '2022-11-28',
+        },
+      },
+    )
+  },
```

Comment:
```suggestion
  scheduleAndRun: async (scheduler_res_path, runName) => octokit.request(
      'POST /repos/{owner}/{repo}/actions/workflows/{workflow_id}/dispatches',
      {
        owner,
        repo,
        workflow_id,
        ref,
        inputs: {
          scheduler_res_path,
          run_name: runName,
        },
        headers: {
          'X-GitHub-Api-Version': '2022-11-28',
        },
      },
    ),
```
I.e. if your function in arrow syntax only consists of an expression, you don't need to open a block with an explicit return.  As the function is marked asynchronous, there also is no need to `await` the result.  This applies `rerunFailed` below, too.